views/views_aggregator_plugin_style_table.inc

```diff
@@ -728,6 +735,64 @@ class views_aggregator_plugin_style_table extends views_plugin_style_table {
     return is_array($rendered_value) ? backdrop_render($rendered_value) : $rendered_value;
   }
 
+  /**
+   * Format a raw timestamp according to the supplied date handler settings.
+   *
+   * @param object $field_handler
+   * @param int $value
+   *
+   * @return string, date formatted according to Views handler settings
+   *
+   * Note: this was taken in part from views_handler_field_date::render($values)
+   */
+  function render_date($field_handler, $value) {
+    $format = $field_handler->options['date_format'];
+    if (in_array($format, array('custom', 'raw time ago', 'time ago', 'today time ago', 'raw time hence', 'time hence', 'raw time span', 'time span', 'raw time span', 'inverse time span', 'time span'))) {
+      $custom_format = $field_handler->options['custom_date_format'];
+    }
+
+    $timezone = !empty($field_handler->options['timezone']) ? $field_handler->options['timezone'] : NULL;
+    $time_diff = REQUEST_TIME - $value; // will be positive for a datetime in the past (ago), and negative for a datetime in the future (hence)
+    switch ($format) {
+      case 'raw time ago':
+        return format_interval($time_diff, is_numeric($custom_format) ? $custom_format : 2);
+      case 'time ago':
+        return t('%time ago', array('%time' => format_interval($time_diff, is_numeric($custom_format) ? $custom_format : 2)));
+      case 'today time ago':
```

Comment:
The format `'today time ago'` was removed from Backdrop. Please remove this `case` fully from your PR.